CTU_typst_template.qmd

```diff
@@ -28,13 +29,81 @@ $$
 
 ## Subsection
 
-Subsection
+Subsection: Use citations like @ref1 and @ref2.
+
+# Table options
+
+```{r, echo=T}
```

Comment:
i would remove the echo from this line as it's also used in the #| lines below... :)